website/docusaurus.config.js

```diff
@@ -90,16 +90,30 @@ const config = {
                     src: 'img/logo/logo-no-text.svg',
                 },
                 items: [
+                    {
+                        type: 'docsVersionDropdown',
+                        position: 'right',
+                        dropdownItemsAfter: [
+                            {
+                                href: 'https://databend-2pvgtjnrv-databend.vercel.app',
+                                label: 'v0.7.172-nightly',
```

Comment:
Why v0.7.172-nightmy?

Reply:
The latest tag for `v0.7` track.

Reply:
I think we should not add 0.7 or 0.6 here, because we don't want user to use them, suggestions keep one 0.8 and next version

Reply:
So you wanna to have `v0.8.0-nightly` and `Next` here?

Reply:
What about the latest version: v0.8.38-nightly and the next?

Reply:
They are nearly the same (only one day's commits)

Reply:
Or we just defer this issue until we released `v1.0` and established a release policy?

Reply:
OK, let's consider enable the version after 0.9 released, many users are using v0.8 :)